foundation/foundation-archetype/test-project-archetype.sh

```diff
@@ -99,72 +99,99 @@ echo "{
 
 # ExampleDataDelivery.json
 echo "{
-      	\"name\": \"ExampleDataDeliveryPipeline\",
-      	\"package\": \"${package}\",
-      	\"type\": {
-      		\"name\": \"data-flow\",
-      		\"implementation\": \"data-delivery-spark\"
-      	}
+        \"name\": \"ExampleDataDeliveryPipeline\",
+        \"package\": \"${package}\",
+        \"type\": {
+          \"name\": \"data-flow\",
+          \"implementation\": \"data-delivery-spark\"
+        },
+        \"steps\": [
+          {
+            \"name\": \"Ingest\",
+            \"type\": \"synchronous\",
+            \"inbound\": {
```

Comment:
I: This step is new. `inbound` + `recordType` is the condition that causes the bug.